### PR TITLE
Add .NET approvers to CODEOWNERS

### PR DESCRIPTION
**Pull request by chalin**
- Contributes to #1320

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 # content owners
 content/en/docs/instrumentation/cpp/     @open-telemetry/docs-approvers @open-telemetry/cpp-maintainers
+content/en/docs/instrumentation/net/     @open-telemetry/docs-approvers @open-telemetry/dotnet-approvers @open-telemetry/dotnet-instrumentation-approvers
```
Comment (chalin) [inline .github/CODEOWNERS:20]: @austinlparker @reyang: are you ok with the addition of these teams, or would you prefer that I instead add the `-maintainers` teams? 

If you agree with the use of these teams, I'll address the access errors reported by GitHub.

Comment (reyang) [reply .github/CODEOWNERS:20]: I'm okay +1 👍

**Review by reyang — APPROVED** [review]
LGTM, thanks @chalin!

File: .github/CODEOWNERS (hunk 1 above)
Comment (reyang) [inline .github/CODEOWNERS:20]: minor: consider putting `net` after `java` (alphabetic sorting)

Comment (chalin) [reply .github/CODEOWNERS]: @austinlparker +1'd out-of-band

Comment (chalin) [reply .github/CODEOWNERS]: Done

**Comment by chalin** [comment]
Updated permissions of the two newly added teams. GH checks are green:

> image

**Pull request merged**